src/binance.mjs

```diff
@@ -116,9 +121,22 @@ export class BinanceAccess {
         this.accounts = new Map();
     }
 
+    async isAvailable() {
+        //Documenation is weak on whether this returns anything other than '{}'.
+        //Monitor this response to see if we get anything else.
+        console.log(this.send(BinanceCommands.available));
+
+        return true;
+    }
+
     async apiCommand(cmd, params={}, account=null) {
         if (cmd === undefined) return false;
 
+        //Check we have a valid connection.
+        if(!this.isAvailable()) {
```

Comment:
I don't think we should do this here, for a number of reasons.
1. Any error from the ping request will be the same as the error you will get from whatever other call you're making, so you're not really gaining anything by making this additional request. Errors will come in the form of an exception which we're not doing much to catch at this point but I want to observe error conditions before coming up with a strategy. Currently happy with any errors ending the script.
2. Some API commands are time sensitive, so adding an additional request in here will increase latency.

Reply:
Fair enough. I was unaware of their limiting and i think the cost of 1 + x for each call is to much. This can instead be part of the init of binanceAccess instead.

Reply:
Well BinanceAccess init already does a binanceInfo call which is equivalent to the ping except you get useful information back. That's the thing - the ping doesn't really give you anything over what other commands do, except those other commands have useful effects.